modules/host/net.go

```diff
@@ -0,0 +1,49 @@
+package host
+
+import (
+	"net"
+
+	"github.com/NebulousLabs/Sia/encoding"
+)
+
+type rpcID [8]byte
```

Comment:
It will become more necessary as the renter/host network semantics become more complex (e.g. monitored connections)